openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```diff
@@ -632,19 +633,19 @@
     "restore-entities-error": "Error while restoring {{entity}}",
     "restore-entities-success": "{{entity}} restored successfully",
     "scopes-comma-separated": "Scopes value comma separated",
```

Comment:
Add the Scopes value, separated by commas.

I hope that's the same meaning being conveyed.